onnx/reference/ops/op_cum_sum.py

```diff
@@ -14,7 +14,7 @@ def _run(self, x, *axis, exclusive=None, reverse=None):  # type: ignore
         if axis is None:  # type: ignore
             if reverse or exclusive:
                 raise NotImplementedError("reverse=1 or exclusive=1 not implemented")
-            return (np.cumsum(x),)
+            return np.cumsum(x, dtype=x.dtype)
```

Comment:
The runtime expects a tuple as an output even if it contains one element. This should break one unit test.

Reply:
Thanks for looking over it!

That is a good catch, however it leaves me wondering if this code is accessed actually since tests are still passing. The onnx documentation states:
> - axis (heterogeneous) - T2:
> A 0-D tensor. Must be in the range [-rank(x), rank(x)-1]. Negative value means counting dimensions from the back.

Doesn't this make the `axis` parameter non-optional? The latest commit removes the case `axis is None` and the tests are still passing on my side.

Reply:
You are right. You can safely remove that section.